.github/workflows/issue_pm_ai.yml

```diff
@@ -0,0 +1,114 @@
+name: Issue PM AI
+
+on:
+  issues:
+    types: [opened]
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  respond:
+    # Only run for issues opened in the main repository (not forks)
+    if: github.repository == 'internetarchive/openlibrary'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Load PM AI Instructions
+        id: pm
+        run: |
+          {
+            echo "INSTRUCTIONS<<EOF"
+            cat .github/prompts/issue_pm_instructions.md
+            echo "EOF"
+          } >> "$GITHUB_OUTPUT"
```

Comment:
This doesn't look like it will work.  In case of bugs, job output documentation is [here](https://docs.github.com/en/actions/how-tos/write-workflows/choose-what-workflows-do/pass-job-outputs).